### PR TITLE
Add some resources

### LINKED ISSUE #53: Add article: "Graphics by Squares: a Gfx-rs Tutorial"
https://suhr.github.io/gsgt/

### LINKED ISSUE #73: Add article
https://aochagavia.github.io/blog/rocket---a-rust-game-running-on-wasm/

### LINKED ISSUE #153: Add: Game server in 150 lines of Rust
https://medium.com/@buterajay/game-server-in-150-lines-of-rust-ce1782199907

### LINKED ISSUE #237: Add rofrol/awesome-wgpu-rs
https://github.com/rofrol/awesome-wgpu-rs

> A curated list of wgpu-rs code and resources.

### PR DESCRIPTION
**Pull request by ozkriff**
Closes #53 (_"Add article: "Graphics by Squares: a Gfx-rs Tutorial"_")
Closes #73 (_"Add article"_)
Closes #153 (_"Add: Game server in 150 lines of Rust"_)
Closes #237 (_"Add rofrol/awesome-wgpu-rs"_)

Also, changes the category of "Roguelike Tutorial in Rust + tcod" to "tutorial".

### CODE REVIEW

File: content/resources/data.toml [hunk 1]
```diff
@@ -10,12 +10,30 @@ description = "A curated list of Rust code and resources, inspired by other awes
...
+
 [[items]]
 name = "Developing a game with Rust beginners"
 description = "Some thoughts on tools and libraries"
 categories = ["articles"]
 homepage_url = "//lukaskalbertodt.github.io/2016/09/01/developing-a-game-with-Rust-beginners-some-thoughts-on-tools-and-libraries.html"
 
+[[items]]
+name = "Game Server in 150 lines of Rust"
+description = "This tutorial will show you how to write asimple game server"
```
Comment (17cupsofcoffee) [inline content/resources/data.toml]:
```suggestion
description = "This tutorial will show you how to write a simple game server"
```

File: content/resources/data.toml [hunk 2]
```diff
@@ -10,12 +10,30 @@ description = "A curated list of Rust code and resources, inspired by other awes
 categories = ["lists"]
 homepage_url = "//github.com/kud1ing/awesome-rust"
 
+[[items]]
+name = "Awesome wgpu"
+description = "A curated list of wgpu code and resources"
+categories = ["lists"]
+homepage_url = "github.com/rofrol/awesome-wgpu"
```
Comment (17cupsofcoffee) [inline content/resources/data.toml]: This either needs a `//` or a `https://` at the front - it's currently getting resolved as a relative link (another thing to add to the CI check, maybe?)

```suggestion
homepage_url = "//github.com/rofrol/awesome-wgpu"
```

File: content/resources/data.toml [hunk 3]
```diff
@@ -10,12 +10,30 @@ description = "A curated list of Rust code and resources, inspired by other awes
...
+name = "Game Server in 150 lines of Rust"
+description = "This tutorial will show you how to write asimple game server"
+categories = ["tutorials"]
+homepage_url = "//medium.com/@buterajay/game-server-in-150-lines-of-rust-ce1782199907"
+
+[[items]]
+name = "Graphics by Squares"
+description = "A gfx-rs tutorial"
+categories = ["tutorials"]
+homepage_url = "//https://suhr.github.io/gsgt"
```
Comment (17cupsofcoffee) [inline content/resources/data.toml]: This works, but it's a little confusing to have `//` *and* `https://`. You only need one :)

```suggestion
homepage_url = "//suhr.github.io/gsgt"
```

**Comment by ozkriff** [comment]
Fixed!

**Pull request merged**